src/main/resources/interface-specification.yml

```diff
@@ -4170,6 +4243,10 @@ paths:
               $ref: '#/components/schemas/PurposeVersionState'
             default: []
           explode: false
+        - in: query
+          name: excludeDraft
+          schema:
+            type: boolean  
```

Comment:
This should not be exposed to the frontend.
The flag is set by the endpoint when calling the process
```suggestion
```